notebooks/embedding/20240327-is-dino.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.            numpy_array = last_hidden_states.numpy()
</pre><p>Also worth considering running the DCT on the flattened dimension in this function. I'm actually not sure how large the output is here, but doing the transform and filtering to the top-n coefficients will save us quite a bit of space. There's some of this data in snakeclef already pre-embedded, so maybe its worth to do the frequency response plot there. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-kaggle-clef/plantclef-2024/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Mostly a comment for myself, but if you have good intuition about this kind of thing it's worth exploring in depth. Otherwise I think the notebook looks fine as-is for a work in progress kind of thing, once you have it successfully running.